### PR TITLE
Create Settings Page ViewModel

### PR DESCRIPTION
**Pull request by soutolucas**

### CODE REVIEW

File: PontoFacil/PontoFacil/ViewModels/SettingsPageViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            set { SetProperty(ref _profile, value); }
+        }
+
+        public DelegateCommand SaveCommand { get; private set; }
+        public DelegateCommand NotifyCommand { get; private set; }
+
+        public SettingsPageViewModel()
+        {
+            Profile = new Profile();
+            Time = new List<int> { 1, 2 };
```
Comment (gmojunior) [inline PontoFacil/PontoFacil/ViewModels/SettingsPageViewModel.cs]: Create a ENUM for this.

**Review by gmojunior — APPROVED** [review]

File: PontoFacil/PontoFacil/Views/SettingsPage.xaml [hunk 2]
```diff
@@ -17,25 +17,28 @@
         <StackPanel>
             <TextBlock FontSize="25" Margin="0, 0, 0, 20 ">Configurações</TextBlock>
             <TextBlock>Nome do usuário</TextBlock>
-            <TextBox></TextBox>
+            <TextBox Text="{x:Bind ViewModel.Profile.Name, Mode=TwoWay}"></TextBox>
```
Comment (lhohenrique) [inline PontoFacil/PontoFacil/Views/SettingsPage.xaml]: Use Binding instead x:Bind to access the Profile property directly

File: PontoFacil/PontoFacil/Views/SettingsPage.xaml.cs [hunk 3]
```diff
@@ -25,6 +15,9 @@ public sealed partial class SettingsPage : Page
         public SettingsPage()
         {
             this.InitializeComponent();
+            ApplicationView.GetForCurrentView().SetPreferredMinSize(new Size(500, 1000));
         }
+
+        public SettingsPageViewModel ViewModel { get { return (SettingsPageViewModel)DataContext; } }
```
Comment (lhohenrique) [inline PontoFacil/PontoFacil/Views/SettingsPage.xaml.cs]: Using Binding at the xaml file this row could be removed

**Review by lhohenrique — CHANGES_REQUESTED** [review]

File: PontoFacil/PontoFacil/Views/SettingsPage.xaml.cs (hunk 3 above)
Comment (soutolucas) [reply PontoFacil/PontoFacil/Views/SettingsPage.xaml.cs]: It's necessary for get properties in ViewModel when use x:bind.

File: PontoFacil/PontoFacil/Views/SettingsPage.xaml (hunk 2 above)
Comment (soutolucas) [reply PontoFacil/PontoFacil/Views/SettingsPage.xaml]: x:Bind have a best performance when compare with binding.

**Pull request merged**